server/controllers/comment.js

```diff
@@ -1,5 +1,6 @@
 import models from "@models";
 import { httpResponse, serverError } from "@helpers/http";
+import { sendCommentNotification } from "./notification";
```

Comment:
please can you use our module resolve.

Reply:
Module resolver does not affect files that are in the same folder. You can take a look in the `index.js` file in the middlewares folder.